cirq/contrib/two_qubit_gates/math_utils.py

```diff
@@ -128,27 +130,56 @@ def _kak_equivalent_vectors(kak_vec) -> np.ndarray:
     return np.reshape(out, out.shape[:-4] + (192, 3))
 
 
-def KAK_vector_infidelity(k_vec_a: np.ndarray,
+def kak_vector_infidelity(k_vec_a: np.ndarray,
                           k_vec_b: np.ndarray,
                           ignore_equivalent_vectors: bool = False
                          ) -> np.ndarray:
-    """Minimum entanglement infidelity between two KAK vectors. """
+    r"""The locally invariant infidelity between two KAK vectors.
+
+    This is the quantity
```

Comment:
Missing the end of this sentence :)

Reply:
The sentence continues in the equation below. I added a line-break for clarity.